multiqc/modules/ngsderive/ngsderive.py

```diff
@@ -455,3 +481,79 @@ def add_junctions_data(self):
             [the documentation](https://stjudecloud.github.io/ngsderive/subcommands/junction_annotation/).""",
             plot=bargraph.plot([bardata, bardata], cats, pconfig),
         )
+
+    def add_endedness_data(self):
+        # Write data to file
+        self.write_data_file(self.endedness, "ngsderive_endedness")
+
+        general_data = {}
+        rpt_present = False
+        for sample, endedness_data in self.endedness.items():
+            general_data[sample] = {
+                "endedness": endedness_data.get("Endedness"),
+            }
+            if "Reads per template" in endedness_data:
+                rpt_present = True
+
+        general_headers = OrderedDict()
+        general_headers["endedness"] = {
+            "title": "Predicted Endedness",
+            "description": "Predicted library endedness from ngsderive",
+        }
+        self.general_stats_addcols(general_data, general_headers)
+
+        headers = OrderedDict()
+        headers["FyLn"] = {
+            "title": "f+l- count",
+            "description": "Number of reads with the 'first in template' FLAG set and the 'last in template' FLAG unset",
+            "format": "{:,.d}",
```

Comment:
> For the metrics showing read counts it's nice to use the `config.read_count_prefix`, `config.read_count_desc`, and `config.read_count_multiplier`, like here: https://github.com/ewels/MultiQC/blob/master/multiqc/modules/verifybamid/verifybamid.py#L178-L186

I can't tell from that code snippet what the variable `config` is. From context, I'm gathering this is a way to toggle units( i.e. "millions of reads")? I think for this table we want to prevent that. The sanity check this is performing (for Paired-End data) is that the _exact_ same number of "first" reads and "last" reads exist in the file. Hence we don't want to lose any precision. We want the full number of reads displayed, not an approximation.

Reply:
Got it, that makes sense! Agree with that. 
I'm mostly concerned with the readability of large numbers, but fixing the `"{:,d}"` format will add spaces between decimal groups, so it will be readable anyway.